src/components/thumbnail/class-work.tsx

```diff
@@ -48,17 +99,21 @@ export class ClassWorkComponent extends BaseComponent<IProps, {}> {
                   <div>{user && user.fullName}</div>
                 </div>
               </div>
+              : null
             );
           })}
+          </div>
         </div>
-      </div>
-    );
+      );
+    } else {
+      return null;
+    }
   }
 
   private handlePublicationClicked = (publication: DocumentModelType) => {
     const {ui} = this.stores;
     return (e: React.MouseEvent<HTMLDivElement>) => {
-      this.stores.ui.rightNavDocumentSelected(publication);
+      ui.rightNavDocumentSelected(publication);
```

Comment:
I adjusted this because we pulled `ui` out into a local above so I assumed we intended to use it.  However, in a case like this where we're only retrieving a single value from the store it might be better to leave it as is and don't bother with making an extra local var.

Reply:
👍